package.json

```diff
@@ -29,13 +29,14 @@
     "@spark-ui/theme-utils": "*"
   },
   "devDependencies": {
-    "@spark-ui/cli-utils": "*",
     "@babel/core": "^7.20.12",
+    "@clack/prompts": "^0.2.1",
```

Comment:
Shouldn't be this be a dependency of the cli package ?

Reply:
I'm adding them :) On brikke we have a warning for when a dependency is missing in local package, maybe we can introduce that also :)
![Capture d’écran 2023-02-15 à 11 37 09](https://user-images.githubusercontent.com/2033710/219004219-8440a4a4-d3b7-413d-bc2b-3179933530a0.png)